backstory/src/main/java/com/google/sps/data/PerspectiveAPIKey.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public class PerspectiveAPIKey {
+  /**
+   * Returns the constant representing API key (or "foo" to hide the key)
+   */
+  public static String getKey() {
+    return "foo";
```

Comment:
Perhaps something more descriptive to illustrate the withholding of the key like "None Set" would be more appropriate.

Reply:
I went with "foo" because that was what Joe had done up above. I wouldn't be opposed to doing something else; however, since the implementation is purely internal (the result of this method will never go further than a Java servlet), I thought it would be okay. @ykjchen Would you suggest still using "foo" or was that not an important element of your suggestions above?